src/Operations.jl

```diff
@@ -953,12 +953,12 @@ function any_package_not_installed(manifest::Manifest)
     return false
 end
 
-function build(ctx::Context, uuids::Set{UUID}, verbose::Bool)
+function build(ctx::Context, uuids::Set{UUID}, verbose::Bool; allow_reresolve::Bool=true)
```

Comment:
Shall we default to `allow_reresolve = false`? Since this use case (build-only deps) seems pretty uncommon.

```suggestion
function build(ctx::Context, uuids::Set{UUID}, verbose::Bool; allow_reresolve::Bool=false)
```